tests/foreman/cli/test_host.py

```diff
@@ -607,6 +607,26 @@ def test_positive_create_with_puppet_class_name(self):
             [puppet['name'] for puppet in host_classes]
         )
 
+    @tier1
+    def test_positive_create_with_openscap_proxy_id(self):
+        """Check if host can be created with OpenSCAP Proxy id
+
+        :id: 3774ba08-3b18-4e64-b07f-53f6aa0504f3
+
+        :expectedresults: Host is created and has OpenSCAP Proxy assigned
+
+        :CaseImportance: Medium
+        """
+        openscap_proxy = Proxy.list({
+            'search': 'url = https://{0}:9090'.format(settings.server.hostname)
+        })[0]
+
```

Comment:
Do we going to have more scenarios for 'openscap-proxy' as the same case we had for puppet proxy earlier and that was placed in a setup function? What do you think?

Reply:
I think we do lack coverage around openscap_proxy (given that this bug wasn't discovered earlier), but I'll leave it up to component owners to add more scenarios. To be honest, right now I'm not even sure what else we should cover - I added this scenario only on request of developer.

So I'd say - leave this in test case for now, move to setup (or even it's own class) as we increase coverage in future.

Reply:
Sure No Problem